hydra/redisgraph_demo1.py

```diff
@@ -0,0 +1,274 @@
+            "prop": str(expect_class)})
+    redis_graph.add_node(node2)
+    create_graph(node, node2, "has_expectclass")
+
+    returned_class = 'obj_supportedclasses_supportedOperation_returns'
+    # put and get method for returns
+    node3 = Node(
+        label="node",
+        alias="operation_returns",
+        properties={
+            "prop": str(returned_class)})
+    redis_graph.add_node(node3)
+    create_graph(node, node3, "has_returns")
+
+    possible_status = 'obj_supportedclasses_supportedOperation_possibleStatus'
+    status_node = Node(
+        label="node",
+        alias="posiblestatus",
+        properties={
+            "prop": possible_status})
+    redis_graph.add_node(status_node)
+    create_graph(node, status_node, "has_status")
+    # put and get method for possible status in class possible_status:
+
+    possibleStatus(status_node)
+
+
+def supported_property(supportedprop_node):
+
+    title = 'obj_supportedclasses_supportedProperty_title'
+    # put and get method for supp_prop title
+    node = Node(
+        label="node",
+        alias="property_title",
+        properties={
+            "prop": title})
+    redis_graph.add_node(node)
+    create_graph(supportedprop_node, node, "has_title")
+
+    properties = 'obj_supportedclasses_supportedProperty_property'
+    # put and get method for properties class Property
+    property_links = "all links of property"
+    property_node = Node(
+        label="node",
+        alias="property",
+        properties={
+            "prop": property_links})
+    redis_graph.add_node(property_node)
+    create_graph(node, property_node, "has_property")
+
+    Property(property_node)
+
+    required = 'obj_supportedclasses_supportedProperty_requires'
+    # put and get method for reuires
+    node2 = Node(label="node", alias="requires", properties={"prop": required})
+    redis_graph.add_node(node2)
+    create_graph(node, node2, "has_requires")
+
+    readonly = 'obj_supportedclasses_supportedProperty_readonly'
+    # put and get method for readonly
+    node3 = Node(
+        label="node",
+        alias="readonly",
+        properties={
+            "prop": str(readonly)})
+    redis_graph.add_node(node3)
+    create_graph(node, node3, "is_readonly")
+
+    writeonly = 'obj_supportedclasses_supportedProperty_writeonly'
+    # put and get method for writeonly
+    node3 = Node(
+        label="node",
+        alias="writeonly",
+        properties={
+            "prop": str(writeonly)})
+    redis_graph.add_node(node3)
+    create_graph(node, node3, "is_writeonly")
+
+
+def possibleStatus(status_node):
+
+    status_code="obj_supportedclasses_supportedOperation_possiblestatus_statusCode"
+    # put and get method
+    node = Node(
+        label="node",
+        alias="status_code",
+        properties={
+            "prop": str(status_code)})
+    redis_graph.add_node(node)
+    create_graph(status_node, node, "has_status")
+
+
+def Property(property_node):
```

Comment:
Please add proper description annotations to each function using `""" """`. Also, make sure your code is PEP8 complaint.